taxi/views.py

```diff
@@ -87,3 +89,32 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+
+
+@login_required
```

Comment:
You can put this logic into post method of `CarDetailView` instead. The reason is that we don't need to create extra views and url endpoints to handle adding/deleting driver.
The implementation will look something like this:

1. Update form instead on car detail template that will send value 'Add' or 'Delete' to post method to the cardetailview.
2. You override post method in this view and catch those values that you received in if condition and add/remove driver to car through related fields (like you did in methods below).
3. You will need to figure out how to get car and user in the view's post method to set related fields.